crates/turbopack-cli-utils/src/trace_writer.rs

```diff
@@ -0,0 +1,98 @@
+// This is inspired by tracing-appender non_blocking, but allows writing a owned
+// Vec<u8> instead of a reference, and uses a unbounded channel to avoid slowing
+// down the application.
```

Comment:
```suggestion
// This is inspired by tracing-appender non_blocking, but allows writing an owned
// Vec<u8> instead of a reference, and uses an unbounded channel to avoid slowing
// down the application.
```

Reply:
This could be a doc comment on the struct directly.